### PR TITLE
build(deps): upgrade deps and fix vulns snyk found

### PR DESCRIPTION
**Pull request by matt-boris**
> This patch upgrades dependencies to their latest minor/patch versions. Doing so fixes a handful of vulns found within Snyk and docker scans.

See vulns by running `docker scan taskcluster/taskcluster:v44.16.4`.

### CODE REVIEW

File: services/web-server/src/main.js [hunk 1]
```diff
@@ -132,6 +132,7 @@ const load = loader(
           tracing: true,
           plugins: [ApolloServerPluginDrainHttpServer({ httpServer })],
           csrfPrevention: true,
+          cache: 'bounded',
```
Comment (matt-boris) [inline services/web-server/src/main.js:135]: https://github.com/apollographql/apollo-server/blob/main/CHANGELOG.md#v390

**Review by petemoore — APPROVED** [review]
Thanks Boris!

File: services/web-server/src/main.js (hunk 1 above)
Comment (matt-boris) [reply services/web-server/src/main.js:136]: https://github.com/taskcluster/taskcluster/pull/5535/files#diff-34ea755b1bde6647b339b37ff4f3b5f6152613dc4a7a269dbea38dc337d9bda2R135-R136

**Review by petemoore — APPROVED** [review]
Awesome, thanks Boris!

**Pull request merged**